Add v1.36 Release Schedule Document (#2936)
kubernetes/sig-release: releases/release-1.36/README.md

---
title: "Kubernetes v1.36 Release Information"
weight: 98
slug: "release"
aliases: [ "/release" ]
description: |
  Information regarding the current release cycle including important dates,
  Release Team contact information, tracking spreadsheets and more!
---

# Kubernetes v1.36

#### Links

* [This document](https://git.k8s.io/sig-release/releases/release-1.36/README.md)
* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.36/release-team.md)
* [Meeting Minutes](https://rel.k8s.io/v136/releasemtg) (members of [release-team@] receive meeting invites)
* [v1.36 Release Calendar](https://rel.k8s.io/release-team-cal)
* Contact: [#sig-release](https://kubernetes.slack.com/archives/C2C40FMNF) on
  slack, [release-team](mailto:release-team@kubernetes.io) on e-mail

> * [Internal Contact Info](https://rel.k8s.io/contacts) (accessible only to members of [release-team@])

Reviewer: The link needs a small correction. Apologies for previously suggesting the broken link.

```suggestion
* [Internal Contact Info](https://rel.k8s.io/v136/contacts) (accessible only to members of [release-team@])
```

> * [Internal Contact Info](https://bit.ly/k8s136-contacts) (accessible only to members of [release-team@])

Reviewer: We might need to update this **bit.ly** link to an equivalent **_rel.k8s.io_** entry in [links.md (here)](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.36/links.md), for consistency.

```suggestion
* [Internal Contact Info](https://rel.k8s.io/contacts) (accessible only to members of [release-team@])
```

description: |
  Information regarding the current release cycle including important dates,
  Release Team contact information, tracking spreadsheets and more!
---

# Kubernetes v1.36

#### Links

* [This document](https://git.k8s.io/sig-release/releases/release-1.36/README.md)
* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.36/release-team.md)
* [Meeting Minutes](https://rel.k8s.io/v136/releasemtg) (members of [release-team@] receive meeting invites)
* [v1.36 Release Calendar](https://rel.k8s.io/k8s-release-cal)
* Contact: [#sig-release](https://kubernetes.slack.com/archives/C2C40FMNF) on
  slack, [release-team](mailto:release-team@kubernetes.io) on e-mail
* [Internal Contact Info](https://bit.ly/k8s136-contacts) (accessible only to members of [release-team@])

#### Tracking docs

* [Enhancements Tracking Board](https://rel.k8s.io/v136/enhancements)
* [Feature blog Tracking Board](https://rel.k8s.io/v136/feature-blogs)
* [Bug Triage Tracking Board](https://rel.k8s.io/v136/bugtriage)
* [CI Signal Project Board & Issue Tracking](https://rel.k8s.io/v136/cisignal)
* [Retrospective Document](https://rel.k8s.io/v136/retro)

> * [kubernetes/kubernetes v1.36 milestone](https://github.com/kubernetes/kubernetes/milestone/68)

Author: Thanks for the catch!

#### Guides

* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)

## Summary

The v1.36 release cycle is as follows:

- **Monday 12th January 2026**: Week 1 — Release cycle begins
- **Wednesday 4th February 2026 (AoE) / Thursday 5th February 2026, 12:00 UTC**: Week 4 — [Production Readiness Freeze][Production Readiness Freeze]
- **Wednesday 11th February 2026 (AoE) / Thursday 12th February 2026, 12:00 UTC**: Week 5 — [Enhancements Freeze][Enhancements Freeze]
- **Thursday 26th February 2026 (AoE) / Friday 27th February 2026, 12:00 UTC**: Week 7 — [Feature blog freeze][Feature blog freeze]

> - **Thursday 19th March 2026 (AoE) / Friday 20th March 2026, 12:00 UTC**: Week 10 — [Code Freeze][Code Freeze] and [Test Freeze][Test Freeze]

Author: Looking at the Code Freeze discussion in [v1.35 retro](https://docs.google.com/document/d/1HIEV02LmdmernZtwCRI44Zv4egv1plof2udmUmG2fFs/edit?tab=t.0#heading=h.ukbaidczvy3r), I can see the original intention was to allow timely exception handling even when the code freeze is right before KubeCon. I have moved this to Wednesday AoE in this cycle with 06e7957d and 6ccb4aad, which now makes all the key dates to be Wednesday AoE (except for feature blog opt-in)

| v1.36.0-alpha.1 released                                      | Branch Manager                | Wednesday 4th February 2026                                                 | week 4   |                                                                          |
| **Begin [Production Readiness Freeze]**                       | Enhancements Lead             | Wednesday 4th February 2026 (AoE) / Thursday 5th February 2026, 12:00 UTC   | week 4   |                                                                          |
| **Begin [Enhancements Freeze]**                               | Enhancements Lead             | Wednesday 11th February 2026 (AoE) / Thursday 12th February 2026, 12:00 UTC | week 5   |                                                                          |
| v1.36.0-alpha.2 released                                      | Branch Manager                | Wednesday 18th February 2026                                                | week 6   |                                                                          |

> | Docs deadline — Open placeholder PRs                          | Docs Lead                     | Thursday 19th February 2026                                                 | week 6   |                                                                          |

Reviewer: This "Docs deadline — Open placeholder PRs" deadline one week after the enhancements freeze may be tight. Can we consider moving this to Week 7, Thursday 26th February? Even though it is just raising a placeholder PR, it usually takes multiple outreach efforts from Docs team.

| Brace Yourself, Code Freeze is Coming                         | Comms / Release Signal        | Monday 23rd February 2026                                                   | week 7   |                                                                          |

> | **Begin [Feature blog freeze] - Open placeholder PRs**        | Comms Lead                    | Thursday 26th February 2026 (AoE) / Friday 27th February 2026, 12:00 UTC    | week 7   |                                                                          |

Author: This was updated along with the Docs deadline update a5fdd1a7

> | Call for Code Freeze [Exceptions][Exception]                  | Lead                          | Monday 2nd March 2026                                                       | week 8   |                                                                          |

Reviewer: Would it make sense to move the **_Call for Code Freeze Exceptions_** to Week 9, Monday 9 March 2026, so that it happens one week before the Code and Test Freeze in Week 10?

| Start final draft of Release Notes                            | Docs Lead                     | Wednesday 8th April 2026                                                    | week 13  |                                                                          |
| **[Docs Freeze]**                                             | Docs Lead                     | Wednesday 8th April 2026 (AoE) / Thursday 9th April 2026, 12:00 UTC         | week 13  |                                                                          |
| Release blog ready to review                                  | Comms / Docs                  | Thursday 9th April 2026 (AoE) / Friday 10th April 2026, 12:00 UTC           | week 13  |                                                                          |
| v1.36.0-rc.1 released                                         | Branch Manager                | Tuesday 14th April 2026                                                     | week 14  |                                                                          |

> | Release Notes complete — reviewed & merged to [k/k][k/k] repo | Docs Lead                     | Wednesday 22nd April 2026                                                   | week 15  |                                                                          |

Reviewer: Historically all the release schedules mention merging this into k/k, however in practice the Docs (and release notes) raise and merge the release notes PR in this [k/sig-release repository](https://github.com/kubernetes/sig-release/pulls?q=is%3Apr+Release+Notes+final+draft+is%3Aclosed). I am not fully sure what is meant by ‘Release Notes complete-reviewed and merged to k/k', this might just be my lack of understanding here 😅

Author: It's very possible we didn't pay too much attention 🫠
Completing Release Notes is surely worth mentioning, but we may be able to drop the mention of the repo, as that's quite specific to Docs only?

| **v1.36.0 released**                                          | Branch Manager                | Wednesday 22nd April 2026                                                   | week 15  |                                                                          |
| Release blog published                                        | Comms Lead                    | Wednesday 22nd April 2026                                                   | week 15  |                                                                          |

> | [Thaw]                                                        | Branch Manager                | Thursday 23rd April 2026                                                    | week 15  |                                                                          |

Reviewer: **Thaw** should happen on the same day as the release, we may want to align this date.

```suggestion
| [Thaw]                                                        | Branch Manager                | Wednesday 22nd April 2026                                                    | week 15  |                                                                          |
```

| Feature blog publication starts                               | Comms Lead                    | Thursday 23rd April 2026                                                    | week 15  |                                                                          |

> ### Team focused view

Reviewer: I see that the new ‘Team focused view’ section has been added to the schedule, and my first thought is that it feels verbose. Several rows repeat items like start of the release cycle, schedule finalized, and team finalized, and many milestones appear the same across multiple team tables. As a result, it increases the overall document length.

Author: What I actually want is to have a single table, and allow anyone to filter dynamically in the view. I don't think that's something we can do with GitHub rendering, though. We could have a calendar view per subteam, but that feels a bit too much.

I agree this is a bit too noisy. The idea was to make the timeline more accessible to all, but it's quite cumbersome to update all now with some changes happened above. I've dropped this section entirely with 1acaa73b, and I just keep the commit reference here for now.
https://github.com/rytswd/sig-release/blob/e41b9a8ab394b40eeb1f64e343529790cb77cb0a/releases/release-1.36/README.md#team-focused-view

* [Feature blog Tracking Board](https://rel.k8s.io/v136/feature-blogs)
* [Bug Triage Tracking Board](https://rel.k8s.io/v136/bugtriage)
* [CI Signal Project Board & Issue Tracking](https://rel.k8s.io/v136/cisignal)
* [Retrospective Document](https://rel.k8s.io/v136/retro)
* [kubernetes/kubernetes v1.36 milestone](https://github.com/kubernetes/kubernetes/milestone/68)

#### Guides

* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)

## Summary

The v1.36 release cycle is as follows:

- **Monday 12th January 2026**: Week 1 — Release cycle begins
- **Wednesday 4th February 2026 (AoE) / Friday 5th February 2026, 12:00 UTC**: Week 4 — [Production Readiness Freeze][Production Readiness Freeze]
- **Wednesday 11th February 2026 (AoE) / Friday 12th February 2026, 12:00 UTC**: Week 5 — [Enhancements Freeze][Enhancements Freeze]
- **Thursday 26th February 2026 (AoE) / Friday 27th February 2026, 12:00 UTC**: Week 7 — [Feature blog freeze][Feature blog freeze]

> - **Thursday 5th March 2026 (AoE) / Friday 6th March 2026, 12:00 UTC**: Week 8 — [Code Freeze][Code Freeze] and [Test Freeze][Test Freeze]

Author: We could even push out to Week 10, which was the case with v1.33. I took most of the timeline from v1.35 for this draft. It was largely affected by KubeCon taking place in Week 9, so I think Week 9 or 10 should be feasible with v1.36.

> - **Thursday 5th March 2026 (AoE) / Friday 6th March 2026, 12:00 UTC**: Week 8 — [Code Freeze][Code Freeze] and [Test Freeze][Test Freeze]

Reviewer 2: Please do. ~1.33 and~ 1.35 had historically short time from KEP freeze to code freeze.

See https://docs.google.com/spreadsheets/d/1r7nv5qeSPbU2OieEht6U1it5m4QeS3Iv7a83ujbs7kU/edit?resourcekey=0-vc4oKe2uijpmpoeyjgRfEQ&gid=0#gid=0

> - **Thursday 5th March 2026 (AoE) / Friday 6th March 2026, 12:00 UTC**: Week 8 — [Code Freeze][Code Freeze] and [Test Freeze][Test Freeze]

Reviewer 2: sorry, 1.35 only - same spreadsheet was used in 1.33 to push it to week 10 :)

> - **Thursday 5th March 2026 (AoE) / Friday 6th March 2026, 12:00 UTC**: Week 8 — [Code Freeze][Code Freeze] and [Test Freeze][Test Freeze]

Author: This has been updated to Week 10 with c0dbfa70
